src/discussions/posts/post/PostFooter.jsx

```diff
@@ -62,6 +62,9 @@ function PostFooter({
             </span>
           </>
         )}
+      {post.unreadCommentCount && post.unreadCommentCount > 0 && post.commentCount > 1 ? (
+        <Badge variant="light">{post.unreadCommentCount} new</Badge>
```

Comment:
This string should be translatable, please add a corresponding message and use the intl machinery to load it.

Reply:
@xitij2000 Okay. I will update this.